integration_tests/e2e/education-support-plan/refuse-plan/refuseEducationSupportPlan.cy.ts

```diff
@@ -5,21 +5,22 @@ import PlanCreationScheduleExemptionReason from '../../../../server/enums/planCr
 import { patchRequestedFor } from '../../../mockApis/wiremock/requestPatternBuilder'
 import { urlEqualTo } from '../../../mockApis/wiremock/matchers/url'
 import { matchingJsonPath } from '../../../mockApis/wiremock/matchers/content'
+import AuthorisationErrorPage from '../../../pages/authorisationError'
 
 context('Refuse an Education Support Plan', () => {
   const prisonNumber = 'H4115SD'
 
   beforeEach(() => {
     cy.task('reset')
-    cy.task('stubSignIn')
-    cy.signIn()
     cy.task('getPrisonerById', prisonNumber)
     cy.task('stubGetEducationSupportPlanCreationSchedules', { prisonNumber })
     cy.task('stubUpdateEducationSupportPlanCreationStatus', prisonNumber)
   })
 
   it('should be able to navigate directly to the refuse Education Support Plan page', () => {
     // Given
+    cy.task('stubSignIn', { roles: ['ROLE_SAN_EDUCATION_MANAGER'] }) // user has the role that gives them permission to refuse ELSPs)
```

Comment:
Now that the "refuse plan" journey has the correct RBAC applied (ie. the other changes in this PR), the cypress tests for refusing a plan need to change to make sure the user has the correct role